istioctl/cmd/injector-list.go

```diff
@@ -301,6 +303,22 @@ func podCountByRevision(pods []v1.Pod, expectedRevision string) map[string]revis
 	return retval
 }
 
+func extractRevisionFromPod(pod *v1.Pod) string {
+	rev := pod.GetLabels()[label.IoIstioRev.Name]
+	if rev != "" {
+		return rev
+	}
+	statusAnno, ok := pod.GetAnnotations()[annotation.SidecarStatus.Name]
```

Comment:
why not just always use the status annotation? It is the real source of truth, not the label

Reply:
@howardjohn Fixed. Thanks